spec/support/api_responses_helper.rb

```diff
@@ -15,7 +16,10 @@ def api_success_response(response = {}, headers = {})
   end
 
   def jsonapi_success_response(type, response = {}, headers = {})
-    formatted_response = format_json_api_response(type, response)
+    formatted_response = format_json_api_response(
```

Comment:
No particular objection but not understanding the reason to vertically format method args? Line length is the normal driver but thats not an issue here?

Reply:
This was an accident. I have a plugin called RevJ that does a reverse join on joined lines and I must have accidentally done this :)